### PR TITLE
Add Websocket prefix settings

### PR DESCRIPTION
**Pull request by yzx-28**

### CODE REVIEW

**Comment by yzx-28** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by pandadtdyy — CHANGES_REQUESTED** [review]
prefix is used in front of the URL

File: packages/ui-default/components/message/index.page.ts [hunk 1]
```diff
@@ -36,8 +36,8 @@ const onmessage = (msg) => {
   }).show();
   return true;
 };
-
-const url = new URL('/home/messages-conn', window.location.href);
+const WsPrefix = UiContext.ws_prefix === '/' ? window.location.href : UiContext.ws_prefix;
+const url = new URL('/home/messages-conn', WsPrefix);
```
Comment (pandadtdyy) [inline packages/ui-default/components/message/index.page.ts]: const url = new URL(`${UiContext.ws_prefix}home/messages-conn`, window.location.href);

**Review by pandadtdyy — CHANGES_REQUESTED** [review]

File: packages/ui-default/components/message/index.page.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ const onmessage = (msg) => {
   return true;
 };
 
-const url = new URL('/home/messages-conn', window.location.href);
+const url = new URL(`${UiContext.ws_prefix}home/messages-conn`, window.location.href);
```
Comment (undefined-moe) [inline packages/ui-default/components/message/index.page.ts:40]: also for record_conn, record_detail_conn, etc

**Review by undefined-moe — CHANGES_REQUESTED** [review]

**Review by undefined-moe — APPROVED** [review]

**Review by pandadtdyy — APPROVED** [review]

**Pull request merged**